modules/lightning_features/lightning_core/src/Access/AdministrativeRoleCheck.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\lightning_core\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\Routing\Route;
+
```

Comment:
Whoops, misread the original comment. This class implements the new _is_administrator access check, which is being used in a couple of routing files also updated in this PR.